ocfweb/account/register.py

```diff
@@ -37,8 +42,14 @@ def request_account(request: HttpRequest) -> Union[HttpResponseRedirect, HttpRes
     status = 'new_request'
 
     existing_accounts = search.users_by_calnet_uid(calnet_uid)
+    eligible_group_signatories = groups_by_student_signat(calnet_uid)
 
-    if existing_accounts and calnet_uid not in TESTER_CALNET_UIDS:
+    existing_group_accounts = {}
+    for group_oid in list(eligible_group_signatories.keys()):
+        if group_by_oid(group_oid) and group_oid not in map(operator.itemgetter(0), TEST_GROUP_ACCOUNTS):
```

Comment:
why do we need to do `group_by_oid(group_oid)` here? isn't the group already guaranteed to exist from line 45?

also ik its less functional but can we just use a list comprehension? my brain hurt